manuscript/draft.Rmd

```diff
@@ -75,20 +75,20 @@ ecological models capture the key processes governing natural systems
 especially important, due to biodiversity's central role in conservation
 planning and its sensitivity to anthropogenic effects [@cardinale2012;
 @diaz2015; @tilman2017]. High-profile studies forecasting large biodiversity
-declines over the coming decades have played a large role in shaping ecologists'
-priorities [as well as those of policymakers; e.g. @ipcc2014], but it is
-inherently difficult to evaluate such long-term predictions before the projected
-biodiversity declines have occurred.
+declines over the coming decades have played a large role in shaping the
+priorities of ecologists and policymakers [e.g. @ipcc2014], but it is inherently
+difficult to evaluate such long-term predictions prior to the predicted decline
+in diversity.
 
 Previous efforts to predict future patterns of terrestrial species richness, and
 diversity more generally, have focused primarily on building species
 distributions models [SDMs; @thomas2004; @thuiller2011; @urban2015]. In general,
 these models describe individual species' occurrence patterns as functions of
 the environment. Given forecasts for environmental conditions, these models can
-predict where each species will occur in the future. These species-level 
-predictions are then combined ("stacked") to generate forecasts for species 
+predict where each species will occur in the future. These species-level
+predictions are then combined ("stacked") to generate forecasts for species
 richness [e.g. @calabrese2014]. Alternatively, models that directly relate
-spatial patterns of species richness to environment conditions have been
+spatial patterns of species richness to environment conditions are available
 developed and generally perform equivalently to stacked SDMs [@algar2009;
```

Comment:
This should either be "have been developed" (the original) or "are available" (i.e., the "developed" needs to be cut as well). I prefer the original but don't care that much.